### PR TITLE
fix vj circulate_date

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

File: kirin/gtfs_rt/model_maker.py [hunk 1]
```diff
@@ -128,8 +129,30 @@ def _get_navitia_vjs(self, trip, data_time):
                                              .format(t=vj_source_code,
                                                      s=since,
                                                      u=until))
-
-        return [model.VehicleJourney(nav_vj, since.date()) for nav_vj in navitia_vjs]
+        VJs = []
+        for nav_vj in navitia_vjs:
+            # Now we compute the real cirluate_date of VJ from since, until and vj's first stop_time
```
Comment (TeXitoi) [inline kirin/gtfs_rt/model_maker.py]: cirluate_date -> typo

File: kirin/gtfs_rt/model_maker.py [hunk 2]
```diff
@@ -128,8 +129,30 @@ def _get_navitia_vjs(self, trip, data_time):
...
+            # We do this to prevent cases like pass midnight when [since, until] is too large
+            if since.date() == until.date():
```
Comment (TeXitoi) [inline kirin/gtfs_rt/model_maker.py]: doesn't work

File: kirin/utils.py [hunk 3]
```diff
@@ -79,3 +81,22 @@ def make_rt_update(data, connector):
     model.db.session.add(rt_update)
     model.db.session.commit()
     return rt_update
+
+
+def get_timezone(stop_time):
```
Comment (TeXitoi) [inline kirin/utils.py]: Please add a TODO: we must use the coverage timezone, not the stop_area timezone, as they can be different. We don't have this information now but we should have it in the near future

File: kirin/core/model.py [hunk 4]
```diff
@@ -73,6 +73,7 @@ class VehicleJourney(db.Model):
     """
     id = db.Column(postgresql.UUID, default=gen_uuid, primary_key=True)
     navitia_trip_id = db.Column(db.Text, nullable=False)
+    # in local timezone
```
Comment (pbougue) [inline kirin/core/model.py]: minor: easier to understand if it's on the concerned line (so end of line below)

File: kirin/gtfs_rt/model_maker.py [hunk 5]
```diff
@@ -133,7 +136,7 @@ def _get_navitia_vjs(self, trip, data_time):
                                   u=until))
             record_internal_failure('gtfs-rt', 'missing vj')
             return []
-
+
```
Comment (pbougue) [inline kirin/gtfs_rt/model_maker.py]: minor: useless space

File: kirin/gtfs_rt/model_maker.py [hunk 6]
```diff
@@ -144,8 +147,41 @@ def _get_navitia_vjs(self, trip, data_time):
                                   ))
             record_internal_failure('gtfs-rt', 'duplicate vjs')
             return []
+       
+        nav_vj = navitia_vjs[0]
+
+        # Now we compute the real circulate_date of VJ from since, until, data_time and vj's first stop_time
```
Comment (pbougue) [inline kirin/gtfs_rt/model_maker.py]: dat`e`_time

**Pull request merged**